### PR TITLE
Icons in list

### PR DESCRIPTION
**Pull request by yougotdirked**
-added icons to the list with items
-icons are stored in .svg files
-added necessary css classes

### CODE REVIEW

File: template.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
         img-src 'self';
         script-src 'self' 'sha256-Ae3eHf6iH3HmkxQ5VMpiwB2ZZoFXcpdPgXGAyxoGFBk=';
         style-src 'self';
+        object-src 'self';
+        frame-src 'self';
```
Comment (LucaScorpion) [inline template.html:13]: Are these (still) necessary? The `object`s are no longer used, and we don't use iframes, right?

**Comment by LucaScorpion** [comment]
With these assets now in separate files, does it make sense to also replace the inline `<svg>` elements?

**Comment by yougotdirked** [comment]
I think its wise to leave the svg tags inline as much as possible as it gives way more control. I used  tags because there was no way around this, unfortunately. A case can be made that it should be unified, though - but the image tag is the "lowest common denominator" i guess

**Comment by LucaScorpion** [comment]
@yougotdirked I see. The consistency was indeed my main point/worry :sweat_smile: Since with this setup we define the SVGs in 2 places (inline and in a file). I tried playing around with it locally, and there it does seem to work fine if we use an img tag instead of the inline svg. The `.block-image svg {` CSS selector then needs to be changed to `.block-image img {`, but that seems to be all that's needed. So that approach would have a (slight) preference from me.

**Review by LucaScorpion — APPROVED** [review]

**Pull request merged**